src/platform/x86_64/apic.zig

```diff
@@ -2,6 +2,34 @@ const os = @import("root").os;
 const std = @import("std");
 const builtin = @import("builtin");
 
+
+var lapic: *volatile [0x100]u32 = undefined;
+
+const IA32_APIC_BASE = @import("regs.zig").MSR(u64, 0x0000001B);
```

Comment:
sneak_case

Reply:
CONSTANTS_ARE_IN_UPPERCASE (afaik)

Reply:
How do you feel about the constants at the bottom?

Reply:
its not exactly a constant. Its MSR you write to and read from

Reply:
@N00byEdge Until we get a predefined order of things in the file, its fine I guess

Reply:
Fixed